src/actions/guides/getting-started/installing.cr

```diff
@@ -29,54 +163,63 @@ class Guides::GettingStarted::Installing < GuideAction
     [forego](https://github.com/ddollar/forego#installation),
     or [foreman](https://github.com/ddollar/foreman#installation).
 
-    > By  default Lucky creates a `Procfile.dev` that  defines  what processes should be started when running `lucky dev`.
+    > By default Lucky creates a `Procfile.dev` that  defines  what processes should be started when running `lucky dev`.
     > You can modify the `Procfile.dev` to start other processes like running
     > background jobs.
 
-    ### Postgres database
+    ## Postgres database
+
+    ### 1. Install Postgres
 
     Lucky uses Postgres for its database. Install Postgres ([macOS](https://postgresapp.com)/[Others](https://wiki.postgresql.org/wiki/Detailed_installation_guides))
 
-    ### Debian and Fedora dependencies
+    ### 1a. (macOS only) Ensure Postgres CLI tools installed
 
-    * Debian (Ubuntu should be similar): run `apt-get install libc6-dev libevent-dev libpcre2-dev libpng-dev libssl1.0-dev libyaml-dev zlib1g-dev`
-    * Fedora (28): run `dnf install glibc-devel libevent-devel pcre2-devel openssl-devel libyaml-devel zlib-devel libpng-devel`. libpng-devel is for Laravel Mix
+    If you're using [Postgres.app](https://postgresapp.com) on macOS make sure
+    [Postgres CLI tools](https://postgresapp.com/documentation/cli-tools.html) are installed
 
-    ### For building assets (skip if building APIs)
+    ```plain
+    sudo mkdir -p /etc/paths.d &&
+      echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp
+    ```
 
-    * [Node](https://nodejs.org/en/download/). Requires at least v6
-    * [Yarn](https://yarnpkg.com/lang/en/docs/install/)
+    There are other installation methods available in [Postgres CLI tools docs](https://postgresapp.com/documentation/cli-tools.html)
+
+    ### 2. Ensure Postgres CLI tools installed
+
+    ```plain
+    psql --version
+    ```
 
-    ### Dependencies for browser tests
+    Should return `psql (PostgreSQL) 12.x` or higher
```

Comment:
Avram works on 9.6 or later, but maybe we just say 10.x or higher? 12 is really new, so some people may think "oh, I have to upgrade postgres to run this?"

Reply:
Good point! I think 10.x sounds good